sops.go

```diff
@@ -137,6 +137,19 @@ func (tree Tree) Encrypt(key []byte, cipher DataKeyCipher) (string, error) {
 	return fmt.Sprintf("%X", hash.Sum(nil)), nil
 }
 
+// FindKey retrieves the data key from the first MasterKey in the Metadata's KeySources that's able to return it.
+func (metadata Metadata) FindKey() ([]byte, error) {
```

Comment:
Sorry for being pedantic on the naming here, but I think it should be renamed `GetDataKey`. Find implies we're searching for a location when we're really iterating over encrypted records, decrypting them, and returning a key.

The term `Data Key` is also important because of the distinction between master (kms/pgp) and data (file) keys.